src/main/java/net/minecraftforge/fluids/Fluid.java

```diff
@@ -110,6 +110,7 @@
      *
      * Generally this is associated with negative density fluids.
      */
+    @Deprecated
```

Comment:
should isGaseous() be removed entirely in 1.12, renamed, or left as it is in this PR? That affects what the proper migration strategy is. It gets used in I think two places right now but it wouldn't be hard to swap those out for manual density sign checks